README.md

```diff
@@ -1,213 +1,158 @@
 # Zipline
 
 This library streamlines using Kotlin/JS libraries from Kotlin/JVM and Kotlin/Native programs. It
-makes it possible to do continuous deployment of mobile apps, just like we do for servers and web
+makes it possible to do continuous deployment of mobile code, just like we do for servers and web
 applications.
 
 It'd be simpler to do continuous deploys via the App Store & Play Store! But that process is too
 slow. Even if we could release via these channels every day, we can’t guarantee that user’s devices
-will take the updates immediately. For Cash App we support apps that haven’t been updated for a full
-year.
+will take the updates immediately.
```

Comment:
This whole paragraph belongs with the second sentence above because it's part of the same single example usage.